postgres/src/main/java/org/jdbi/v3/postgres/PostgresPlugin.java

```diff
@@ -146,6 +147,7 @@ public void customizeJdbi(Jdbi jdbi) {
         jdbi.registerArrayType(new ByteaArrayType());
 
         jdbi.registerColumnMapper(new JavaTimeMapperFactory());
+        jdbi.registerColumnMapper(new InternetMapperFactory());
```

Comment:
this is registered with the core mappers in the BuiltinMapperFactory, so this should not be needed.